packages/@webex/http-core/test/unit/spec/interceptors/http-status.js

```diff
@@ -43,6 +43,18 @@ describe('http-core', () => {
 
           assert.isRejected(interceptor.onResponse({}, response));
         });
+        it('resolves on appapi redirect error', () => {
```

Comment:
We should have a test like:
```
        it('rejects when locus redirect is not intended', () => {
          const response = {
            statusCode: 404,
            body: {
              errorCode: 2000001,
            },
          };

          assert.isRejected(interceptor.onResponse({}, response));
        });
        ```
        
        but for appAPI with a different code